### PR TITLE
Total available liquidity fix

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/sdk/src/modules/FusePools.ts [hunk 1]
```diff
@@ -132,6 +132,7 @@ export function withFusePools<TBase extends FuseBaseConstructor>(Base: TBase) {
         comptroller,
         name,
         totalLiquidityNative,
+        totaAvailablelLiquidityNative,
```
Comment (peetzweg) [inline packages/sdk/src/modules/FusePools.ts:135]:
```suggestion
        totalAvailablelLiquidityNative,
```

File: packages/sdk/src/modules/FusePools.ts [hunk 2]
```diff
@@ -104,13 +102,15 @@ export function withFusePools<TBase extends FuseBaseConstructor>(Base: TBase) {
...
+          Number(utils.formatUnits(asset.liquidity)) * Number(utils.formatUnits(asset.underlyingPrice));
 
         totalSuppliedNative += asset.totalSupplyNative;
         totalBorrowedNative += asset.totalBorrowNative;
 
-        asset.liquidityNative =
-          Number(utils.formatUnits(asset.liquidity)) * Number(utils.formatUnits(asset.underlyingPrice));
+        asset.liquidityNative = assetLiquidity;
 
+        totaAvailablelLiquidityNative += asset.isBorrowPaused ? 0 : assetLiquidity;
```
Comment (peetzweg) [inline packages/sdk/src/modules/FusePools.ts:113]:
```suggestion
        totalAvailablelLiquidityNative += asset.isBorrowPaused ? 0 : assetLiquidity;
```

File: packages/sdk/src/modules/FusePools.ts [hunk 3]
```diff
@@ -41,6 +41,7 @@ export function withFusePools<TBase extends FuseBaseConstructor>(Base: TBase) {
       ).map(filterOnlyObjectProperties);
 
       let totalLiquidityNative = 0;
+      let totaAvailablelLiquidityNative = 0;
```
Comment (peetzweg) [inline packages/sdk/src/modules/FusePools.ts:44]:
```suggestion
      let totalAvailablelLiquidityNative = 0;
```

File: packages/sdk/src/modules/FusePools.ts [hunk 4]
```diff
@@ -59,12 +60,9 @@ export function withFusePools<TBase extends FuseBaseConstructor>(Base: TBase) {
       for (let i = 0; i < assets.length; i++) {
         const asset = assets[i];
 
+        const isBorrowPaused: boolean = await comptrollerContract.callStatic.borrowGuardianPaused(asset.cToken);
```
Comment (peetzweg) [inline packages/sdk/src/modules/FusePools.ts:63]: Why the refactor? 
We are creating a promise array to run all of these requests in parallel. 
Having this one separate will just halt the rest of the execution till it's done.

And we don't need the value to proceed, nothing depends on it. I would prefer the previous version to this.

The rest of this function is also using this style, why do it differently for `comptrollerContract.callStatic.borrowGuardianPaused` ? :shrug:

File: packages/sdk/src/modules/FusePools.ts (hunk 3 above)
Comment (peetzweg) [reply packages/sdk/src/modules/FusePools.ts:44]: typo

File: packages/sdk/src/modules/FusePools.ts (hunk 4 above)
Comment (peetzweg) [reply packages/sdk/src/modules/FusePools.ts:63]: Never mind, it is used. Just seeing it now. 🤦

File: packages/sdk/src/modules/FusePools.ts (hunk 1 above)
Comment (peetzweg) [reply packages/sdk/src/modules/FusePools.ts:135]: typo

Comment (carlomazzaferro) [reply packages/sdk/src/modules/FusePools.ts:135]: and there's still a typo (doule L) haha. Will fix

File: packages/ui/components/pages/Fuse/FusePoolPage/BorrowList.tsx [hunk 5]
```diff
@@ -43,6 +43,7 @@ export const BorrowList = ({ assets, borrowBalanceFiat, comptrollerAddress }: Bo
   );
   const unBorrowableAssets = assets.filter((asset) => asset.isBorrowPaused);
 
+  // eslint-disable-next-line no-console
```
Comment (peetzweg) [inline packages/ui/components/pages/Fuse/FusePoolPage/BorrowList.tsx:46]: no console in the next line, so could be removed.
```suggestion
  // eslint-disable-next-line no-console
```

**Review by peetzweg — APPROVED** [review]

**Pull request merged**